### ISSUE TITLE
logos are distorted

### ISSUE THREAD

**Issue by dret** [u1i/catalysts#1]
it seems all catalyst logos are distorted. maybe they were forced to be square (which they are not). the original logos are here: https://git-ext.ecd.axway.com/catalysts/archive/-/tree/master/logo

**Comment by u1i** [comment]
right. Saw the one on Git-ext - I'm set to speak with NM this week, our new 'brand guidebook' also has versions of the logo in it

**Comment by dret** [comment]
for producing any other colors than the original one, it might be best to start from the original logo linked above, so that we don't use any resolution and get undistorted ones.
